common/com/twitter/intellij/pants/util/PantsUtil.java

```diff
@@ -763,7 +763,15 @@ public static boolean supportExportDefaultJavaSdk(@NotNull final String pantsExe
     }
 
     final String defaultPlatform = exportResult.getJvmPlatforms().getDefaultPlatform();
-    final String jdkName = String.format("JDK from pants %s", defaultPlatform);
+
+    // if defaultPlatform is java8, then jdkName should be 1.8_from_pants
+    String jdkName = "1.x_from_pants";
```

Comment:
do you mean to have the jdkName default to 
`jdkName = String.format("1.x_from_%s", pantsExecutable)`?